### DISCUSSION TITLE
[Esame 31 Agosto 2021] Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#250] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-13 194434](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/15b12177-2aa6-4836-ad70-b1dc6272e1d2)

**Comment by CasuFrost** [comment]
## Punto a )

![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/ce567c0f-0b56-4046-a143-97ba62f5d5f4)
